FrenchFilter/sections/css_extended.txt

```diff
@@ -5,6 +5,9 @@
 ! Good: example.org#?#main > div.row:has(> div.ad)
 ! Bad: example.org##main > div.ad (should be in specific.txt)
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/110189
+numerama.com#?#.pb-3 > div[class]:has(> [class^="Billboard"])
+numerama.com#?#.container > div[class]:has(> [class^="Billboard"])
```

Comment:
As I can see, this `numerama.com##.is-original-antlion` is enough. Have you checked it?